src/transformers/modeling_tf_utils.py

```diff
@@ -69,11 +74,14 @@
 if parse(tf.__version__).minor >= 13:
     from keras import backend as K
     from keras.__internal__ import KerasTensor
+    from keras.engine.base_layer_utils import call_context
```

Comment:
This would break since `keras` 2.13 has changed the import to `keras.src.engine`

See #23663

Reply:
Thanks for the catch, I'll make the fix ASAP!